### PR TITLE
Refactor tabs

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

**Review by mvaivre — COMMENTED** [review]
As always: 👏 

image

Little layout issue

File: apps/desktop-wallet/src/api/apiDataHooks/useSeparateListedFromUnlistedTokens.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { useMemo } from 'react'
+
+import useFTList from '@/api/apiDataHooks/useFTList'
+import { ListedFT, UnlistedToken } from '@/types/tokens'
+
+interface SeparatedTokens<T> {
```
[the remark this answers is not in the document]
Comment (nop33) [reply apps/desktop-wallet/src/api/apiDataHooks/useSeparateListedFromUnlistedTokens.ts:24]: What do you recommend?

File: apps/desktop-wallet/src/features/assetsLists/TokensTabs.tsx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+  const { tabs, isExpanded, toggleExpansion } = useTokensTabs({
+    numberOfNSTs: nstIds.length,
+    ftsTabTitle: `💰 ${t('Address tokens')}`,
+    nstsTabTitle: `❔ ${t('Address unknown tokens')}`,
+    nftsTabTitle: `🖼️ ${t('Address NFTs')}`
+  })
+
+  const [currentTab, setCurrentTab] = useState<TabItem<TokensTabValue>>(tabs[0])
+
+  const _isExpanded = isExpanded || !maxHeightInPx
```
[the remark this answers is not in the document]
Comment (nop33) [reply apps/desktop-wallet/src/features/assetsLists/TokensTabs.tsx:47]: It is removed in the latest PR 👍

File: apps/desktop-wallet/src/api/apiDataHooks/useSeparateListedFromUnlistedTokens.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (nop33) [reply apps/desktop-wallet/src/api/apiDataHooks/useSeparateListedFromUnlistedTokens.ts:24]: `TokensByType`?

[the remark this answers is not in the document]
Comment (mvaivre) [reply apps/desktop-wallet/src/api/apiDataHooks/useSeparateListedFromUnlistedTokens.ts:24]: Yes! sounds good!

**Review by mvaivre — APPROVED** [review]
Thanks for the answers. Seems like a lot has changed in the follow up PRs, so let's move forward!

**Pull request merged**